### PR TITLE
Minor style changes to new argo app sync content 

### PR DESCRIPTION
**Pull request by jonathanglassman**
Minor changes to bring new argo app sync content into gov.uk style

### CODE REVIEW

**Review by theseanything — APPROVED** [review]

**Comment by theseanything** [comment]
Thanks @jonathanglassman!

File: source/manage-app/manage-state/index.html.md.erb [hunk 1]
```diff
@@ -9,13 +9,16 @@ review_in: 6 months
 
 ## Troubleshooting
 
-### Argo Application sync stuck "waiting for completion of hook"
+### Argo application sync stuck waiting for "waiting for completion of hook"
 
-An Argo Application may appear to be stuck syncing forever whilst waiting for the completion of a hook from a Job that not running. In this situation you may need to cancel the sync operation. You can do this from the Argo UI with following steps:
+An Argo application may appear to be stuck syncing forever whilst waiting for the completion of a hook from a Job that is not running. In this situation you may need to cancel the sync operation.
```
Comment (sengi) [inline source/manage-app/manage-state/index.html.md.erb:14]: Application is the name of a type of object in Argo, if that makes a difference. (In a book it'd be bold or fixedwidth or something.)

File: source/manage-app/manage-state/index.html.md.erb [hunk 2]
```diff
@@ -9,13 +9,16 @@ review_in: 6 months
...
 
-1. Choose the Application tile from the list of Applications
-1. Click "Sync Status". A modal pane should slide in from the right of the screen.
-1. Click "Terminate" from the modal to cancel the sync operation
-1. Re-sync the application by clicking "Sync"
+You can cancel the sync operation using the Argo UI.
+
+1. Sign into [Argo](https://argoproj.github.io/).
+1. Choose the app tile from the list of __Applications__.
+1. Select __Sync Status__ to reveal a modal.
```
Comment (sengi) [inline source/manage-app/manage-state/index.html.md.erb:20]: I think the use of the term _modal_ here without context is potentially confusing. It's UI jargon that not all developers/SREs etc. will necessarily be familiar with.

**Review by sengi — APPROVED** [review]

File: source/manage-app/manage-state/index.html.md.erb (hunk 2 above)
Comment (jonathanglassman) [reply source/manage-app/manage-state/index.html.md.erb:20]: Is there a more plain English term? or maybe an appropriate link to a definition?

Comment (sengi) [reply source/manage-app/manage-state/index.html.md.erb:20]: How about this?
```suggestion
1. Select __Sync Status__ to reveal more information about the sync.
```

**Review by sengi — APPROVED** [review]

**Comment by jonathanglassman** [comment]
@sengi @theseanything thanks for the quick reviews and responses

**Pull request merged**